src/index.tsx

```diff
@@ -544,6 +544,9 @@ export class App extends React.Component<{}, AppState> {
             let resizeHandle: ResizeTestType = false;
             let isDraggingElements = false;
             let isResizingElements = false;
+            let someElementIsDragged = false;
```

Comment:
Not clear what's the difference between `someElementIsDragged` & `isDraggingElements`

Reply:
`someElementIsDragged` changes to true if some dragging occured. `isDraggingElements` changes to true if we click to already selected element

Reply:
Maybe there's some conceptual difference, but I don't see any difference in code logic. `someElementIsDragged` is set unconditionally on `isDraggingElements`.

Even if we decided to distinguish between the two (but I still don't see any point now), we should at least rename those variables to reflect it. As it stands those names are identical, which adds to the confusion.

Reply:
Key difference if that `someElementIsDragged` is set on **mousemove** event, when `isDraggingElements` is set on **onmousedown** event.

On the `mouseup` event, we need to ensure that there was no dragging... so the variable `someElementIsDragged` was added to keep that information.

Maybe we can rename them as `isDraggingElements` -> `readyToDrag` and `someElementIsDragged ` -> `dragOccured`?